catalog.yml

```diff
@@ -69,3 +69,10 @@ sources:
     description: Atmospheric radiative profiles during EUREC4A
     driver: intake.catalog.local.YAMLFileCatalog
     metadata: {}
+
+  rv_meteor:
```

Comment:
```suggestion
  Meteor:
```
Let's go for Meteor then. However, I wouldn't change the file names, because I think RV-METEOR or rv-meteor is more descriptive on first glance. Meteor could always be an abbreviation of Meteorology or so. I'll also change the folder name to Meteor.

Reply:
Thats perfect, I also checked the contents of one of the files and they are identifying the `platform_id` as `Meteor` as well :+1:

From the intake catalog perspective, the filenames don't matter, so I am perfectly fine with keeping the filenames as they are.